src/platforms/platform.dom.js

```diff
@@ -162,23 +128,24 @@ module.exports = function(Chart) {
 			}
 		};
 
-		// Let's keep track of this added iframe and thus avoid DOM query when removing it.
-		stub.resizer = createResizer(notify);
+		if (!erd) {
+			erd = elementResizeDetectorMaker({
+				strategy: 'scroll',
+			});
+		}
 
-		node.insertBefore(stub.resizer, node.firstChild);
+		erd.listenTo(node, notify);
+		stub.resizer = true;
```

Comment:
What's the purpose of setting `true` here? In the old code, this was the iframe. Would it be better to set `stub.resizer = erd` ?